src/test/testUtil.ts

```diff
@@ -287,3 +288,14 @@ export async function closeAllEditors(): Promise<void> {
         throw new Error(`The following editors were still open after closeAllEditors():\n${editors.join('\n')}`)
     }
 }
+
+export function testCommand<T extends (...args: any[]) => unknown, U extends any[]>(
+    command: ReturnType<typeof Commands.declare<T, U>>,
+    ...args: U
+): Command<T> {
```

Comment:
```suggestion
): RegisteredCommand<T> {
```
@JadenSimon you mean we could change this right